File: problem_10/problem_10.js
const now = require("performance-now");
const { primeSieve } = require("./prime_sieve")

const problem_10 = () => primeSieve(2000000).reduce((a,b)=>a+b);

let start = now();
let solution = problem_10()
console.log(`${solution} found in ${(now() - start) / 1000} seconds.`)
// 142913828922 found in 0.22473554800000003 seconds.

File: problem_10/problem_10.py
import time
from prime_sieve import prime_sieve

def problem_10():
    return sum(prime_sieve(2000000))

start = time.time()
solution = problem_10()
print(f"{solution} found in {time.time() - start} seconds.")
# 142913828922 found in 0.07372689247131348 seconds.